src/Data/Hours/HoursData.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lits\LibCal\Data\Hours;
+
+use Lits\LibCal\Data;
+
+class HoursData extends Data
+{
+    public ?int $lid = null;
```

Comment:
I think I generally left properties in these classes non-nullable when the API returned a value, and it looks like that happens for everything but $lon and $fn for some reason. I probably could be convinced to allow them to be nullable though.